src/routers/request.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express');
+const userAuthenticationMiddleware = require('../middlewares/user-authentication');
+const requestController = require('../controllers/request');
+
+const router = express.Router();
+
+// GET, POST routes for /api/request/
```

Comment:
I think your route is `/requests` with an "s" in the end. Just update the comment to avoid any confusion.